feat: add `overwrite-pid-file` and `pid-file` (#5972)
influxdata/docs-v2: content/influxdb/v2/reference/config-options.md

---

### overwrite-pid-file

> Overwrite PID file specified with [pid-file](#pid-file) if already present instead of exiting with an error.

Reviewer: ```suggestion
Overwrite the PID file specified by [pid-file](#pid-file) if it exists, instead of exiting with an error.
```

**Default:** `false`

| influxd flag           | Environment variable         | Configuration key    |
| :--------------------- | :--------------------------- | :------------------- |
| `--overwrite-pid-file` | `INFLUXD_OVERWRITE_PID_FILE` | `overwrite-pid-file` |

###### influxd flag
<!--pytest.mark.skip-->

```sh
influxd --overwrite-pid-file
```

###### Environment variable
```sh
export INFLUXD_OVERWRITE_PID_FILE=true
```

###### Configuration file
{{< code-tabs-wrapper >}}
{{% code-tabs %}}
[YAML](#)
[TOML](#)
[JSON](#)
{{% /code-tabs %}}
{{% code-tab-content %}}
```yml
overwrite-pid-file: true
```
{{% /code-tab-content %}}
{{% code-tab-content %}}
```toml
overwrite-pid-file = true
```
{{% /code-tab-content %}}
{{% code-tab-content %}}
```json
{
  "overwrite-pid-file": true
}
```
{{% /code-tab-content %}}
{{< /code-tabs-wrapper >}}

---

### pid-file

> Write PID file to specified path. File is removed when InfluxDB is stopped.

Reviewer: ```suggestion
Write the PID file to the specified path and remove it when InfluxDB stops.
```
